src/test/java/uk/gov/justice/digital/delius/service/ConvictionService_AddOrReplaceOrDeleteCustodyKeyDatesTest.java

```diff
@@ -241,4 +245,138 @@ void willDoNothingWhenNoDatesHaveChanged() {
         verify(eventRepository, never()).save(any());
     }
 
+    @Nested
+    @MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
Surprised you have to set this explicitly - I thought that was the default?

Reply:
It would appear not.

TBH I think it is a sign of a test not structured correctly In this case the parent test assumes it will was aways update a key date hence the need to ignore services never called.